helm/kubernetes-cluster-autoscaler-chart/values.yaml

```diff
@@ -2,6 +2,9 @@ name: cluster-autoscaler
 namespace: kube-system
 serviceType: managed
 
+# Make sure to update the docs at https://github.com/giantswarm/docs if you change the defaults below!
```

Comment:
```suggestion
# Make sure to update the docs at https://github.com/giantswarm/docs/blob/master/src/layouts/shortcodes/autoscaler_utilization_threshold.html if you change the defaults below
```

Reply:
The above URL will be the exact file to modify, once https://github.com/giantswarm/docs/pull/292 is merged.